escrow/packages/frontend/vite.config.ts

```diff
@@ -0,0 +1,40 @@
+import react from '@vitejs/plugin-react';
+import jotaiDebugLabel from 'jotai/babel/plugin-debug-label';
+import jotaiReactRefresh from 'jotai/babel/plugin-react-refresh';
+import './load.envs.ts';
+import { defineConfig } from 'vite';
+import tsconfigPaths from 'vite-tsconfig-paths';
+
+const WHITELIST = ['NODE_ENV', 'PUBLIC_URL'];
```

Comment:
Try to add the `GENESIS_SECRET ` here;

```suggestion
const WHITELIST = ['NODE_ENV', 'PUBLIC_URL', 'GENESIS_SECRET'];
```